presentations.md

```diff
@@ -0,0 +1,29 @@
+# Presentations
+
+Every **1st Saturday** of the month from **3:30 PM EAT** @nairobilug hosts a Meetup with presentations. This document seeks to guide a speaker by giving them the necessary resources they need if they want to contribute a presentations.
+Video recordings of previous meetups are in a [YouTube Playlist](https://youtube.com/playlist?list=PLWYgxwb8YFV1OS4_kWef90dYJvm730x0I)  and text based write-ups are in a [GitHub Repo](https://github.com/nairobilug/presentations).
+
+## Step One
+Have a relevant **Linux** topic you want to discuss about.
```

Comment:
we could just have this as `have a topic you want to discuss about`. This leaves the door open to a lot more interesting conversations.

Reply:
The repo contains the presentation slides